### PR TITLE
ANC PostPartum - Specialty Care Next Steps

### PR DESCRIPTION
**Pull request by anvmn**
#368
#369
#370 
#392

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-08-18 13-28

**Comment by anvmn** [comment]
Peek 2022-08-18 18-24

**Comment by anvmn** [comment]
Selection_1046

**Comment by anvmn** [comment]
Selection_1047

**Comment by anvmn** [comment]
@mariano-dagostino please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -9863,16 +9884,27 @@ translationSet trans =
...
+                , kinyarwanda = Nothing
+                }
+
+            else
+                { english = "Refer patient to ARV services for assessment of ARVs"
+                , kinyarwanda = Nothing
+                }
+
+        PrenatalARVProgramPostpartumHeader ->
+            { english = "This patient was diagnosed with HIV during their pregnancy"
```
Comment (mariano-dagostino) [inline client/src/elm/Translate.elm]: @anvmn Not sure about the use of `their` here. I think it should be `her pregnancy` but not 100% sure.

**Review by mariano-dagostino — APPROVED** [review]
@anvmn Minor comment. Looks good

**Pull request merged**